bin/fin

```diff
@@ -6277,6 +6295,19 @@ case "$1" in
 		shift
 		run_cli "$@"
 		;;
+	cron)
```

Comment:
As discussed let's remove the `cron` command and instead docs below updated to instruct to restart cli after `crontab` file changes

```
Once that file has been created or updated, the `cli` container should be restarted by running `fin restart cli`.
```